src/server/serveractiveobject.cpp

```diff
@@ -18,25 +18,31 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 */
 
 #include "serveractiveobject.h"
+#include <algorithm>
+#include <cstddef>
+#include <cstdio>
 #include <fstream>
+#include <unordered_map>
 #include "inventory.h"
 #include "inventorymanager.h"
 #include "constants.h" // BS
 #include "log.h"
 
-ServerActiveObject::ServerActiveObject(ServerEnvironment *env, v3f pos):
+using SAO = ServerActiveObject;
```

Comment:
Debuggability wasn't much of a concern for me here since I'd usually go by file:line there.

But I agree about the inflated diff so I have undone it.

Reply:
FWIW I think this change is okay but indeed better put in a separate PR